src/content.js

```diff
@@ -155,8 +155,13 @@ export function drawDK(element, keyMap, deadKey) {
   }
   const alt0 = deadKey[keyChars[0]];
   const alt1 = deadKey[keyChars[1]];
-  element.querySelector('.level5').textContent = alt0 || '';
-  element.querySelector('.level6').textContent = altUpperChar(alt0, alt1);
+  if (isDeadKey(alt0)) {
+    element.querySelector('.level5').classList.add('deadKey', 'diacritic');
+    element.querySelector('.level5').textContent = alt0[1];
+  } else {
+    element.querySelector('.level5').textContent = alt0 || '';
+    element.querySelector('.level6').textContent = altUpperChar(alt0, alt1);
+  }
```

Comment:
That’s pretty close, and it works fine with Ergo-L 0.99.2. Two remarks :
- the `$(.level6)` text content is undetermined when `alt0` is a dead key
- the `deadKey` and `diacritic` classes are never removed

So I’d rather suggest something like this : 

```js
  const lvl5 = element.querySelector('.level5');
  const lvl6 = element.querySelector('.level6');
  if (isDeadKey(alt0)) {
    lvl5.classList.add('deadKey', 'diacritic');
    lvl5.textContent = alt0[1];
  } else {
    lvl5.classList.remove('deadKey', 'diacritic');
    lvl5.textContent = alt0 || '';
  }
  lvl6.textContent = altUpperChar(alt0, alt1);
```

To test this, put a dead key in the `1dk` layer on a key that also has a defined symbol on `1dk`<kbd>Shift</kbd>[letter], like for the comma key here : 
```
  ╭╌╌╌╌╌┰─────┬─────┬─────┬─────┬─────┰─────┬─────┬─────┬─────┬─────┰╌╌╌╌╌┬╌╌╌╌╌╮
  ┆ ~   ┃ ! ¡ │ @ ‘ │ # ’ │ $ ¢ │ % ‰ ┃ ^   │ &   │ *   │ «   │ »   ┃ _ – ┆ + ± ┆
  ┆ `   ┃ 1 „ │ 2 “ │ 3 ” │ 4 £ │ 5 € ┃ 6 ¥ │ 7 ¤ │ 8 § │ 9 ¶ │ 0 ° ┃ / ÷ ┆ = ≠ ┆
  ╰╌╌╌╌╌╂─────┼─────┼─────┼─────┼─────╂─────┼─────┼─────┼─────┼─────╂╌╌╌╌╌┼╌╌╌╌╌┤
  ·     ┃ Q   │ C   │ O   │ P   │ W   ┃ J   │ M   │ D   │**   │ F   ┃ {   ┆ }   ┆
  ·     ┃   â │   ç │   œ │   ô │     ┃     │   µ │   _ │***¨ │     ┃ [   ┆ ]   ┆
  ·     ┠─────┼─────┼─────┼─────┼─────╂─────┼─────┼─────┼─────┼─────╂╌╌╌╌╌┼╌╌╌╌╌┤
  ·     ┃ A   │ S   │ E   │ N   │ ; • ┃ L   │ R   │ T   │ I   │ U   ┃ "   ┆ |   ┆
  ·     ┃   à │   é │   è │   ê │ ,*` ┃     │     │   î │   û │   ù ┃ '   ┆ \   ┆
  ╭╌╌╌╌╌╂─────┼─────┼─────┼─────┼─────╂─────┼─────┼─────┼─────┼─────╂╌╌╌╌╌┴╌╌╌╌╌╯
  ┆ >   ┃ Z   │ X   │ ? ¿ │ V   │ B   ┃ :   │ H   │ G   │ Y   │ K   ┃           ·
  ┆ <   ┃     │   ß │ - — │   ñ │     ┃ . … │     │     │     │     ┃           ·
  ╰╌╌╌╌╌┸─────┴─────┴─────┴─────┴─────┸─────┴─────┴─────┴─────┴─────┚ · · · · · ·
```